app/models/drop_topic.rb

```diff
@@ -0,0 +1,34 @@
+class DropTopic  < DeadlineType
+
+   def email_list(assignment_id)
+         emails = mail_sign_up_topic_users(assignment_id)
+         emails
+   end
+
+   def mail_sign_up_topic_users(assignment_id)
+       sign_up_topics = SignUpTopic.where(['assignment_id = ?', assignment_id])
+       assignment = Assignment.find(self.assignment_id)
+       emails =[]
+       for topic in sign_up_topics
+         signed_up_teams = SignedUpTeam.where(['topic_id = ?', topic.id])
+         unless assignment.team_assignment?
```

Comment:
You should check the team size. If you check line 50 in assignment.rb, `assignment.team_assignment?` always returns true.